src/styles/antisnake.scss

```
@@ -0,0 +1,18 @@
+.antisnake {
+  max-width: 1199px;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax($minSizeItem, 1fr));
+  justify-content: center;
```

```suggestion
  display: grid;
  grid-template-columns: repeat(auto-fill, minmax($minSizeItem, 1fr));
  justify-content: center;
  
  max-width: 1199px;
```
Порядок властивостей